src/recovery.rs

```diff
@@ -30,10 +30,9 @@ pub fn recover_partitions(keyspace: &Keyspace) -> crate::Result<()> {
         let partition_name = dirent.file_name();
         let partition_path = dirent.path();
 
-        assert!(
-            dirent.file_type()?.is_dir(),
-            "Found stray file in partitions folder",
-        );
+        if !dirent.file_type()?.is_dir() {
+            log::warn!("Found stray file in partitions folder");
```

Comment:
There needs to be a continue after the log, otherwise, the file be considered a partition and everything breaks.